lib/Dialect/HW/Transforms/LowerHWtoBTOR2.cpp

```diff
@@ -0,0 +1,786 @@
+  const static constexpr StringLiteral input = "input";
+  const static constexpr StringLiteral reset = "reset";
+  const static constexpr StringLiteral output = "output";
+  const static constexpr StringLiteral zero = "zero";
+  const static constexpr StringLiteral one = "one";
+  const static constexpr StringLiteral constStr = "const";
+  const static constexpr StringLiteral constd = "constd";
+  const static constexpr StringLiteral consth = "consth";
+  const static constexpr StringLiteral slice = "slice";
+  const static constexpr StringLiteral uext = "uext";
+  const static constexpr StringLiteral add = "add";
+  const static constexpr StringLiteral sub = "sub";
+  const static constexpr StringLiteral mul = "mul";
+  const static constexpr StringLiteral andStr = "and";
+  const static constexpr StringLiteral orStr = "or";
+  const static constexpr StringLiteral xorStr = "xor";
+  const static constexpr StringLiteral sll = "sll";
+  const static constexpr StringLiteral srl =
+      "srl"; // a.k.a. unsigned right shift
+  const static constexpr StringLiteral sra = "sra"; // a.k.a. signed right shift
+  const static constexpr StringLiteral sdiv = "sdiv";
+  const static constexpr StringLiteral udiv = "udiv";
+  const static constexpr StringLiteral smod = "smod";
+  const static constexpr StringLiteral concat = "concat";
+  const static constexpr StringLiteral notStr = "not";
+  const static constexpr StringLiteral neq = "neq";
+  const static constexpr StringLiteral hwNeq = "ne";
+  const static constexpr StringLiteral ite = "ite";
+  const static constexpr StringLiteral implies =
+      "implies";                                        // logical implication
+  const static constexpr StringLiteral state = "state"; // Register state
+  const static constexpr StringLiteral next =
+      "next"; // Register state transition
+  const static constexpr StringLiteral bad = "bad";
+  const static constexpr StringLiteral constraint = "constraint";
+  const static constexpr StringLiteral ws = " ";  // WhiteSpace
+  const static constexpr StringLiteral nl = "\n"; // NewLine
+
+  /// Field helper functions
+
+  // Checks if a constant of a given size has been declared
+  // If so, its lid will be returned
+  // Otherwise -1 will be returned
+  size_t getConstLID(int64_t val, size_t w) {
+    // Look for the pair
+    for (auto p : constToLIDMap) {
+      // Compare the given value and width to those in the entry
+      if ((val == p.getFirst().first) && (w == p.getFirst().second)) {
+        return p.getSecond();
+      }
+    }
+    // if no lid was found return -1
+    return NO_LID;
+  }
+
+  // Updates or creates an entryfor  a constant of a given size
+  // associating it with the current lid
+  void setConstLID(int64_t val, size_t w) {
+    // Create a new entry for the pair
+    constToLIDMap[std::make_pair<int64_t, size_t>(
+        std::forward<int64_t>(val), std::forward<size_t>(w))] = lid;
```

Comment:
This has a lots of stuff which doesn't do anything.
`constToLIDMap[{val,w}] = lid;`
which is short enough maybe it doesn't need a function?